dashboard/src/components/Config/AppRepoList/AppRepoRefreshAllButton.v2.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { useState } from "react";
+
+import actions from "actions";
+import { CdsButton, CdsIcon } from "components/Clarity/clarity";
+import { useDispatch, useSelector } from "react-redux";
+import { IStoreState } from "shared/types";
+
+export function AppRepoRefreshAllButton() {
+  const [refreshing, setRefreshing] = useState(false);
+  const { repos } = useSelector((state: IStoreState) => state.repos);
+  const dispatch = useDispatch();
+
+  const handleResyncAllClick = async () => {
+    // Fake timeout to show progress
```

Comment:
Why are we faking showing progress when the actual repo is being refreshed? IMO either we shouldn't pretend and instead just use a user info "Repository refresh requested. It may take x minutes to complete.", or it should show actual progress? Again, fine without, just wondering what the longer-term plan is for this "progress".